src/lib/alternatives/UI/alternative_dialog.rb

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2016 SUSE LLC.
+#  All Rights Reserved.
+
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of version 2 or 3 of the GNU General
+#  Public License as published by the Free Software Foundation.
+
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+#  GNU General Public License for more details.
+
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, contact SUSE LLC.
+
+#  To contact SUSE about this file by physical or electronic mail,
+#  you may find current contact information at www.suse.com
+
+require "yast"
+require "ui/dialog"
+
+Yast.import "UI"
+Yast.import "Label"
+
+module Y2Alternatives
```

Comment:
ruby convention for location it camel cased name of class, so here ` Y2Alternatives::Dialog::AlternativeDialog` should be in `y2_alternatives/dialog/alternative_dialog`